patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -1091,9 +1167,28 @@
  					rangedDamage += 0.05f;
  					minionDamage += 0.05f;
 +					*/
++					allCrit += 5;
++					/*
```

Comment:
Why add this here in the middle of a block when you could instead add this to the end since you are added the block end there anyways.